storybook/stories/Stories/UI/ui.stories.js

```diff
@@ -0,0 +1,44 @@
+export default { title: "UI" };
+
+import { initElmStory } from "../../initElmStory.js";
+import Icon from "./Icon.elm";
+import Button from "./Button.elm";
+import Card from "./Card.elm";
+import ErrorCard from "./ErrorCard.elm";
+import Banner from "./Banner.elm";
+import CopyField from "./CopyField.elm";
```

Comment:
I think this one also has a webcomponent associated with it that we'd want to import in the js world.

Reply:
Ok. So are there different ways to use Elm components, other than writing like this?

Reply:
Yeah there's a few Elm modules that have a WebComponent pair to do native JavaScript stuff. In this case its `src/UI/CopyOnClick.js`, which need to be included in the bundle for copy to work (the other examples are `OnClickOutside.js`, `EmbedSvg.js`, `EmbedMermaid.js`, and `EmbedKatex.js` FWIW).

Reply:
I see. I changed the code to import CopyOnClick.js